code/modules/robotics/bot/secbot.dm

```diff
@@ -843,26 +875,27 @@
 		weeoo()
 		process()	// ensure bot quickly responds to a perp
 
-	proc/YellAtPerp()
-		src.point(src.target, 1)
-		src.speak("Level [src.threatlevel] infraction alert!")
+	proc/YellAtPerp(impotent_rage = 0)
```

Comment:
How does this relate to the rest of the PR? I don't see any calls of `YellAtPerp` being added nor modified?

Reply:
Oh yeah, the bot was supposed to occasionally say things while locked inside. I took it out cus it felt kinda dumb and forgot to clean it up. I'll clean it up!

Reply:
Okay I think I got all the impotent rage out.